src/PresentationalComponents/RulesTable/RulesTable.spec.ct.js

```diff
@@ -0,0 +1,642 @@
+    urlParam: 'impact',
+    urlValue: (it) => encodeURIComponent(_.map(it, (x) => IMPACT[x]).join(',')),
+  },
+  likelihood: {
+    selectorText: 'Likelihood',
+    values: Array.from(cumulativeCombinations(Object.keys(LIKELIHOOD))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.map(value, (x) => LIKELIHOOD[x]).includes(it.likelihood),
+    urlParam: 'likelihood',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => LIKELIHOOD[x]).join(',')),
+  },
+  category: {
+    selectorText: 'Category',
+    values: Array.from(cumulativeCombinations(Object.keys(CATEGORIES))),
+    type: 'checkbox',
+    filterFunc: (it, value) =>
+      _.intersection(
+        _.flatMap(value, (x) => CATEGORIES[x]),
+        it.tags
+      ).length > 0,
+    urlParam: 'category',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => CATEGORIES_MAP[x]).join(',')),
+  },
+  status: {
+    selectorText: 'Status',
+    values: STATUS,
+    type: 'radio',
+    filterFunc: (it, value) => {
+      if (value === 'All') return true;
+      else return it.disabled === (value === 'Disabled');
+    },
+    urlParam: 'rule_status',
+    urlValue: (it) => it.toLowerCase(),
+  },
+  impacting: {
+    selectorText: 'Systems impacted',
+    values: Array.from(cumulativeCombinations(Object.keys(IMPACTING))),
+    type: 'checkbox',
+    filterFunc: (it, value) => {
+      if (!value.includes('1 or more') && it.impacted_systems_count > 0)
+        return false;
+      if (!value.includes('None') && it.impacted_systems_count === 0)
+        return false;
+      return true;
+    },
+    urlParam: 'impacting',
+    urlValue: (it) =>
+      encodeURIComponent(_.map(it, (x) => IMPACTING[x]).join(',')),
+  },
+};
+const filterApply = (filters) => applyFilters(filters, filtersConf);
+const filterCombos = [{ impacting: ['1 or more'] }];
+const filterData = (filters = DEFAULT_FILTERS, values = data) =>
+  filter(filtersConf, values, filters);
+
+//Columns should be in a separate file where we will store all the constants???
+export const columns = [
+  {
+    title: intl.formatMessage(messages.name),
+    transforms: [sortable, cellWidth(40)],
+  },
+  {
+    title: intl.formatMessage(messages.modified),
+    transforms: [sortable, cellWidth(10)],
+  },
+  {
+    title: intl.formatMessage(messages.category),
+    transforms: [sortable, cellWidth(10)],
+  },
+  {
+    title: intl.formatMessage(messages.totalRisk),
+    transforms: [sortable, cellWidth(15)],
+  },
+  {
+    title: intl.formatMessage(messages.riskOfChange),
+    transforms: [sortable, cellWidth(15)],
+  },
+  {
+    title: intl.formatMessage(messages.systems),
+    transforms: [sortable, cellWidth(15)],
+  },
+  {
+    title: intl.formatMessage(messages.remediation),
+    transforms: [sortable, cellWidth(15), fitContent],
+  },
+];
+//the default count is 20, you can pass the other number if you need to
+const DEFAULT_ROW_COUNT = 20;
```

Comment:
I remember this was the default value in test utils. Do we need to duplicate it here?

Reply:
You are correct, that's unnecessary